cfgov/v1/atomic_elements/organisms.py

```diff
@@ -100,6 +100,9 @@ class Meta:
         icon = 'mail'
         template = '_includes/organisms/email-signup.html'
 
+    class Media:
+        js = ["email-signup.js"]
```

Comment:
Single quotes added in `https://github.com/cfpb/cfgov-refresh/pull/2482/commits/ad79459ed120111d3e457e2ce40334acb2f96343`. Not sure why we don't use camel-case but I was just following the existing pattern. The referenced file is cfgov/unprocessed/js/routes/on-demand/email-signup.js